### PR TITLE
Split Tables SDK from Storage and changed namespace to CosmosDB

### PR DESCRIPTION
**Pull request by srinathnarayanan**

### CODE REVIEW

File: BreakingChanges.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# Breaking Changes
+
+> See the [Change Log](ChangeLog.md) for a summary of storage library changes.
+
+**Note: This changelog is deprecated starting with version XX.XX.XX, please refer to the ChangeLog.md in each package for future change logs.** 
+
+## Version XX.XX.XX:
+
+### All:
+- The library has been split into 4 different packages:
```
Comment (zezha-msft) [inline BreakingChanges.md]: this needs to be reworded for your release.

**Review by zezha-msft — APPROVED** [review]
It looks fine to me, but obviously I did not look over every file since they're mostly copy-pasting.  :)

File: azure-cosmosdb-table/azure_bdist_wheel.py [hunk 2]
```diff
@@ -512,7 +512,7 @@ def run(self):
         if not self.distribution.install_requires:
             self.distribution.install_requires = []
         self.distribution.install_requires.append(
-            "{}>=2.0.0".format(self.azure_namespace_package))
+            "{}>=0.37.0".format(self.azure_namespace_package))
```
Comment (zezha-msft) [inline azure-cosmosdb-table/azure_bdist_wheel.py]: do no make this change, we purposely keep it 2.0.0.

File: azure-cosmosdb-nspkg/setup.py [hunk 3]
```diff
@@ -35,7 +35,7 @@
 
 setup(
     name='azure-cosmosdb-nspkg',
-    version='2.0.0',
+    version='0.37.0',
```
Comment (zezha-msft) [inline azure-cosmosdb-nspkg/setup.py]: do not make this change

File: azure-cosmosdb-table/azure_bdist_wheel.py (hunk 2 above)
Comment (zezha-msft) [reply azure-cosmosdb-table/azure_bdist_wheel.py]: you can contact Laurent Mazuel to know the specific reason behind this.

Comment (srinathnarayanan) [reply azure-cosmosdb-table/azure_bdist_wheel.py]: pinged him, will revert to 2.0.0

File: azure-cosmosdb-nspkg/setup.py (hunk 3 above)
Comment (srinathnarayanan) [reply azure-cosmosdb-nspkg/setup.py]: will revert to 2.0.0

File: azure-cosmosdb-table/azure_bdist_wheel.py (hunk 2 above)
Comment (zezha-msft) [reply azure-cosmosdb-table/azure_bdist_wheel.py]: cool, what did he say?

**Comment by troydai** [comment]
Are there new features introduced in this PR and is API version changed?

**Comment by zezha-msft** [comment]
@troydai There's no new API, it's the same package that I previously shared with you, under a different namespace. In addition, they merged common into the table package.

**Review by kirankumarkolli — APPROVED** [review]

**Comment by troydai** [comment]
Source code files are encoded differently, most in __UTF-8 with BOM__ while some in __UTF-8__. The mix of encoding makes it difficult to run some automation scripts as well as text comparison tools. __UTF-8 with BOM__ really isn't a standard encoding. Please consider remove BOM on all files and keep files free of BOM.

Ref: https://stackoverflow.com/questions/2223882/whats-different-between-utf-8-and-utf-8-without-bom

File: .gitignore [hunk 4]
```diff
@@ -1,288 +1,59 @@
-## Ignore Visual Studio temporary files, build results, and
```
Comment (moderakh) [inline .gitignore:1]: Why did we update this file? shouldn't we keep existing .gitignore file and maybe add a few more lines to it?

File: BreakingChanges.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Breaking Changes
+
+> See the [Change Log](ChangeLog.md) for a summary of storage library changes.
```
Comment (moderakh) [inline BreakingChanges.md:3]: ">" is that on purpose?

File: LICENSE.txt [hunk 6]
```diff
@@ -0,0 +1,176 @@
+                                 Apache License
```
Comment (moderakh) [inline LICENSE.txt:1]: I think it should be MIT license. Right? all of our other sdks are under MIT license. Any reason you changed this one?

**Review by moderakh — APPROVED** [review]
it would be better if you could have published their code in this git repo first. Then you apply all these changes. That way, git would show you that you only moved some files and maybe edited a few. 

But right now it is hard to know what has changed in this process.

**Pull request merged**